typings/index.flow.js

```diff
@@ -306,8 +306,22 @@ interface Thenable {
   then: (resolve: () => any, reject?: () => any) => any;
 }
 
+type FilterNodeByTypeFunction = (
+  propName: string,
+  propValue: any,
+  node: ReactTestRendererJSON
+) => boolean;
+
+type FormatOptions = {
+  filterProps?: FilterNodeByTypeFunction,
+};
+
+type DebugOptions = {
+  message?: string,
+} & FormatOptions;
```

Comment:
nit: I would go with interface here, but this is purely personal style pref.

```suggestion
interface FormatOptions {
  filterProps?: FilterNodeByTypeFunction,
};

interface DebugOptions extends FormatOptions {
  message?: string,
}
```